optimism/src/keccak/environment.rs

```diff
@@ -425,4 +431,28 @@ impl<F: Field> KeccakEnv<F> {
 
         state_g
     }
+
+    #[allow(dead_code)]
+    /// Returns the list of constraints used in a specific Keccak step
+    pub(crate) fn constraints_of(step: Steps) -> Vec<E<F>> {
```

Comment:
Both `constraints_of` and `lookups_of` are meant to be called to obtain the constraints and lookups corresponding to a given selector for the `HashMap` inside the `Circuit` type for tests. Not sure if I can remove the annotation before I use them in the main though.